test/performance/observed_concurency_test.go

```diff
@@ -132,8 +132,13 @@ func TestObservedConcurrency(t *testing.T) {
 	// Add concurrency metrics
 	metrics := []string{"autoscaler_observed_stable_concurrency", "autoscaler_observed_panic_concurrency", "autoscaler_target_concurrency_per_pod"}
 	for _, metric := range metrics {
-		val := prometheus.RunQuery(context.Background(), logger, promAPI, metric, names)
-		tc = append(tc, createTestCase(float32(val), metric))
+		query := fmt.Sprintf("%s{configuration_namespace=\"%s\", configuration=\"%s\", revision=\"%s\"}", metric, test.ServingNamespace, names.Config, names.Revision)
+		val, err := prometheus.RunQuery(context.Background(), logger, promAPI, query)
+		if err != nil {
+			logger.Infof("Error querying metric %s: %v", metric, err)
```

Comment:
Suppose a metric query fails, how do you figure that out without digging into the logs?

Reply:
tesgrid will report it as 0 and then we can look at the logs to see if there was an error regarding that.

Reply:
why not Errorf?